gitignore.plugin.zsh

```diff
@@ -51,7 +51,8 @@ function _gitignore_template() {
 _gitignore_get_template_list() {
     (for tpath in ${(@s/:/)ZSH_PLUGIN_GITIGNORE_TEMPLATE_PATHS}; do; command find "$tpath" -type f -name "*.gitignore"; done) \
         | command xargs -n 1 basename \
-        | command sed -e 's/.gitignore$//' -e 's/\(.*\)/\L\1/' \
+        | command sed -e 's/.gitignore$//' \
+        | tr "[:upper:]" "[:lower:]" \
```

Comment:
@ajanian, you should add the [precommand modifier](http://zsh.sourceforge.net/Doc/Release/Shell-Grammar.html#Precommand-Modifiers) `command` at the beginning of the line (see https://github.com/voronkovich/gitignore.plugin.zsh/pull/6#issue-226372831). Also I think it's better to use single quotes instead of double ones. IMHO, If you don't need a parameter expansion it's better to use single quotes.